docs/src/_includes/layouts/base.html

```diff
@@ -13,11 +13,19 @@
         {% set page_title = site[hook].title %}
     {% endif %}
 
+    {% set rule_meta = rules_meta[title] %}
     {% set page_title = site.shared.title_format | replace("PAGE_TITLE", page_title) %}
-    {% set cover_image = ["https://", site.hostname, "/icon-512.png" ] | join %}
-    {% set cover_image_alt = site.shared.eslint_logo_alt %}
     {% set page_desc = site.shared.description %}
     {% set relative_page_url = page.url | url | prettyURL %}
+    {% set cover_image = [
+        "https://", site.hostname, "/og",
+        "?title=", title, "&summary=", page_desc,
```

Comment:
I think we should url-encode these two. For example, this wouldn't work as intended for a title that contains a `&` character, like `Configure Plugins & Parsers` (we actually had this title a few versions ago). Page descriptions are currently the same for all pages but we could change it to include rule descriptions which might also contain characters that would break the intended URL.